olp-cpp-sdk-dataservice-write/src/generated/PublishApi.cpp

```diff
@@ -308,8 +308,8 @@ SubmitPublicationResponse PublishApi::CancelPublication(
                                       query_params, header_params, form_params,
                                       nullptr, "application/json", context);
 
-  if (http_response.status != olp::http::HttpStatusCode::NO_CONTENT) {
-    return client::ApiError(http_response.status, http_response.response.str());
+  if (http_response.GetStatus() != olp::http::HttpStatusCode::NO_CONTENT) {
+    return client::ApiError(http_response.GetStatus(), http_response.GetResponseAsString());
```

Comment:
I had impression there's 80 chars limit in the formatting setup. This one looks longer